backend/api/src/db/category.rs

```diff
@@ -61,6 +74,12 @@ order by t.ord
         children: vec![],
         image_count: it.image_count as u64,
         jig_count: it.jig_count as u64,
+        user_scopes: {
+            it.user_scopes
+                .iter()
+                .map(|x| UserScope::try_from(*x).unwrap())
```

Comment:
style comment: would prefer `.into_iter()` to indicate that ownership is being taken over the entire array, rather than `*` which implies copy of the individual elements